splash/qtrender_lua.py

```diff
@@ -313,49 +360,39 @@ def __init__(self, splash, source):
         self.tab = splash.tab
         self.source = escape_js(source)
         self.exceptions = splash.exceptions
+        self.populate_exposed_elements = splash.populate_exposed_elements
 
     @exceptions_as_return_values
     @can_raise
     @emits_lua_objects
+    @create_html_elements_for_nodes
     @decodes_lua_arguments('utf8')
     def __call__(self, *args):
         expr = "eval('('+{func_text}+')')({args})".format(
             func_text=self.source,
-            args=escape_js(*args)
+            args=','.join([arg.element_js if isinstance(arg, HTMLElement) else escape_js(arg) for arg in args])
```

Comment:
could you please extract this line to an utility function? It is used more than once